README.md

```diff
@@ -158,14 +157,14 @@ Please refer to [Shutting down a domain](site/shutdown-domain.md) for informatio
 
 ## Load balancing with the Traefik Ingress controller
 
-The initial Technology Preview release of the operator supports only the Traefik load balancer/Ingress controller.  Support for other load balancers is planned in the future.
-Please refer to [Load balancing with Traefik](site/traefik.md) for information about current capabilities.
+This release of the operator supports only the Traefik load balancer/Ingress controller.  Support for other load balancers is planned in the future.
+Please refer to [Load balancing with Traefik](site/traefik.md) for information about the current capabilities.
 
 [comment]: # (Exporting operator logs to ELK.  The operator provides an option to export its log files to the ELK stack. Please refer to [ELK integration]site/elk.md for information about this capability.)
 
 ## Removing a domain
 
-To permanently remove a domain from a Kubernetes cluster, first shut down the domain using the instructions provided above in the section titled “Shutting down a domain”, then remove the persistent volume claim and the persistent volume using these commands:
+To permanently remove a domain from a Kubernetes cluster, first shut down the domain using the instructions provided in [Shutting down a domain](site/shutdown-domain.md), then remove the persistent volume claim and the persistent volume using these commands:
```

Comment:
The new script delete-weblogic-domain-resources.sh has the ability to delete a specific domain, all domains.  When the script is invoked it cleans all the custom resource definition for the domain. The script has nice documentation, if you need to have this documentation reviewed Tom Barnes should be able to help.

Reply:
So, for Removing a domain, users do not have to first shut it down and then remove the persistent volume and persistent volume claim? Should I just point to the delete-weblogic-domain-resources.sh script? Where is it located (I don't see it in the site directory)?

Reply:
The delete script is kubernetes/delete-weblogic-domain-resources.sh

Reply:
Ok. I'll make reference to it and ask Tom Barnes to review my changes.